tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-inventory-tracker-test.php

```
@@ -0,0 +1,512 @@
+			},
+			10,
+			3
+		);
+
+		// Trigger sync.
+		do_action( WC_Stripe_Agentic_Commerce_Inventory_Tracker::SCHEDULED_ACTION );
+
+		// Pending queue should be cleared.
+		$after = get_option( WC_Stripe_Agentic_Commerce_Inventory_Tracker::PENDING_UPDATES_OPTION, [] );
+		$this->assertEmpty( $after );
+	}
```

PHPMD (2.15.0)

[error] The method test_full_flow_stock_change_to_upload is not named in camelCase. (undefined)

(CamelCaseMethodName)